2-advanced/dubbo-samples-local/case-configuration.yml

```diff
@@ -17,8 +17,22 @@
 
 # local call through injvm, start test container only
 services:
-  dubbo-samples-local-test:
+  local-provider:
```

Comment:
The test failed for lacking `zookeeper` configuration.

https://github.com/apache/dubbo-samples/blob/master/1-basic/dubbo-samples-spring-boot/case-configuration.yml#L18